docs/reference/settings.rst

```diff
@@ -49,6 +58,9 @@ Search
 
 Define a search backend. For a full explanation, see :ref:`wagtailsearch_backends`.
 
+``WAGTAILSEARCH_RESULTS_TEMPLATE``, ``WAGTAILSEARCH_RESULTS_TEMPLATE_AJAX``
```

Comment:
These can be removed (and also from `docs/advanced_topics/add_to_django_project.rst`) - search views were moved into the project template long ago, so there's nothing to override from the wagtailsearch app.

Reply:
Nice catch :) Removing